test/tools/yulopti.cpp

```diff
@@ -265,10 +411,11 @@ int main(int argc, char** argv)
 		po::options_description options(
 			R"(yulopti, yul optimizer exploration tool.
 	Usage: yulopti [Options] <file>
-	Reads <file> as yul code and applies optimizer steps to it,
+	Reads <file> containing a yul object and applies optimizer steps to it,
 	interactively read from stdin.
 	In non-interactive mode a list of steps has to be provided.
 	If <file> is -, yul code is read from stdin and run non-interactively.
+	An <object> flag may be provided, specifying a dotted path to an object in the input.
```

Comment:
```suggestion
	An --object flag may be provided, specifying a dotted path to an object in the input.
```
But I think this is not necessary here. The option description below is already enough.